INSTALL

```diff
@@ -37,9 +35,11 @@ You can customize some settings :
 Among others, you can choose if your users will have automatic_addressbook enabled or disabled by default (use_auto_abook option).
 
 
-Upgrading roundcube to 0.6
+Olders roundcube up to 0.8
 --------------------------
 
+you must use files before commit ca622de3364b996c655eea2e7724ce6e325e9b9d for older versions.
```

Comment:
Prefer reference to plugin version rather than commit id

Reply:
i used commit due that commit included some changes! lasted older version still have minor issues that i'm working on